.github/workflows/ci.yml

```diff
@@ -151,23 +141,92 @@ jobs:
           service_account_key: ${{ secrets.GCP_SA_KEY }}
           export_default_credentials: true
 
-      - name: Configure canary build
-        if: |
-          matrix.kind == 'test' &&
-          matrix.profile == 'release' &&
-          github.repository == 'denoland/deno' &&
-          github.ref == 'refs/heads/main'
-        shell: bash
+      - name: Error on warning
+        # TODO(piscisaureus): enable this on Windows again.
+        if: "!matrix.use_sysroot && !startsWith(matrix.os, 'windows')"
+        run: echo "RUSTFLAGS=-D warnings" >> $GITHUB_ENV
```

Comment:
Oops, I didn't see this.

Why are windows builds getting warnings? I would guess this sysroot change would have no effect on Windows.